samples/serverReports/src/sql/startEvent.sql

```diff
@@ -24,9 +22,11 @@ IF NOT EXISTS (SELECT * FROM sys.dm_xe_sessions  WHERE name = 'ObjectContention'
         ALTER EVENT SESSION [ObjectContention] ON SERVER
         STATE =  START
     END
+SELECT 0 AS RESULTCODE, 'XEvent sessions started for PageContention and ObjectContention' AS RESULTMSG
```

Comment:
It's kind of weird to have the success messages in the SQL script - why not have them in the extension so they can be properly localized?

Reply:
@Charles-Gagnon I took a whack at setting the messages up for localization from the actions, but I skipped localizing the package.json for now. thanks for the feedback!